src/matlabcontrol/MatlabProxyFactoryOptions.java

```diff
@@ -127,6 +140,8 @@ int getPort() {
 		private volatile String _licenseFile = null;
 		private volatile boolean _useSingleCompThread = false;
 		private volatile int _port = 2100;
+		private volatile Writer _inputWriter;
+		private volatile Writer _errorWriter;
 
```

Comment:
Can we add "= null" here?  It's not necessary for correctness, but I think it helps to document that null is the intended default value.